okdata/cli/commands/datasets/boilerplate/config.py

```diff
@@ -16,44 +23,110 @@
 
 available_pipelines = [c.value for c in pipeline_choices if c.value]
 
-boilerplate_questions = [
-    {"type": "text", "name": "title", "message": "Tittel", "validate": TitleValidator},
-    {"type": "text", "name": "description", "message": "Beskrivelse"},
-    {"type": "text", "name": "objective", "message": "Formål"},
-    {
-        "type": "text",
-        "name": "keywords",
-        "message": "Nøkkelord (komma-separert)",
-        "validate": KeywordValidator,
-    },
-    {
-        "type": "select",
-        "name": "accessRights",
-        "message": "Tilgangsnivå",
-        "choices": [
-            Choice("Offentlig", "public"),
-            Choice("Begrenset offentlighet", "restricted"),
-            Choice("Unntatt offentlighet", "non-public"),
-        ],
-    },
-    {"type": "text", "name": "name", "message": "Kontaktperson - navn"},
-    {
-        "type": "text",
-        "name": "email",
-        "message": "Kontaktperson - epost",
-        "validate": SimpleEmailValidator,
-    },
-    {
-        "type": "text",
-        "name": "phone",
-        "message": "Kontaktperson - telefon",
-        "validate": PhoneValidator,
-    },
-    {"type": "text", "name": "publisher", "message": "Utgiver"},
-    {
-        "type": "select",
-        "name": "pipeline",
-        "message": "Prosessering",
-        "choices": pipeline_choices,
-    },
-]
+
+def filter_comma_separated(value):
+    values = next(
+        csv.reader([value], delimiter=",", escapechar="\\", skipinitialspace=True)
+    )
+    return [x.strip() for x in values if x.strip()]
```

Comment:
`strip` trengs vel ikke når `skipinitialspace=True`? Samme spørsmål gjelder også andre steder.

Reply:
Mener du begge? Den "siste" kan nok tas bort ja, ellers har man vel - med f.eks `abc, "def,ghi"   ,jkl,   `:
* `tags = ['abc', 'def,ghi   ', 'jkl', '']`
* `tags = [x for x in tags if x] = ['abc', 'def,ghi   ', 'jkl']`
* `tags = [x.strip() for x in tags if x] = ['abc', 'def,ghi', 'jkl']` <- som jeg tenker er ønskelig

Skal se litt over andre steder; men mener det var noen tilfeller det (i validatorene) bør fungere litt anerledes.

Reply:
Tenkte på begge. For hvordan får man med mellom i `['abc', 'def,ghi ', 'jkl', '']` når `skipinitialspace=True`?

```python
>>> tags = "foo, bar,  baz"
>>> next(csv.reader([tags], skipinitialspace=True))
['foo', 'bar', 'baz']
```

Reply:
Man vil ha med trailing? Sorry, misforstår sikkert her.

Reply:
> Tenkte på begge. For hvordan får man med mellom i `['abc', 'def,ghi ', 'jkl', '']` når `skipinitialspace=True`?
> 
> ```python
> >>> tags = "foo, bar,  baz"
> >>> next(csv.reader([tags], skipinitialspace=True))
> ['foo', 'bar', 'baz']
> ```

Edit: Åja, sorry, så ikke eksempelet ditt. 😛 Da blir det kanskje nødvendig ja.

Reply:
Men da kan kanskje `skipinitialspace=True` droppes, om uansett strippes på begge sider etterpå?

Reply:
Trodde det, og vet ærlig talt ikke hvorfor, men er et case der man med space foran en quoted streng ikke helt får ønsket resultat:

`data = 'abc,"def,ghi",jkl'` ===> `['abc', 'def,ghi', 'jkl']`
`data = 'abc, "def,ghi",jkl'`===> `['abc', '"def', 'ghi"', 'jkl']`

Reply:
Heh, ja, det var en rar cocktaileffekt! Men ja, da er det nok best som det står. Beklager støyet. 😛